cmd/coder/urls.go

```diff
@@ -84,26 +90,35 @@ func (sub createSubCmd) Run(fl *pflag.FlagSet) {
 	if !portIsValid(port) {
 		exitUsage(fl)
 	}
+	portNum, _ := strconv.Atoi(port)
 
 	access = strings.ToUpper(sub.access)
 	if !accessLevelIsValid(access) {
 		exitUsage(fl)
 	}
 
+	name = sub.urlname
+	if name != "" && !devURLNameValidRx.MatchString(name) {
+		flog.Error("update devurl: name must be < 64 chars in length, begin with a letter and only contain letters or digits.")
+		return
```

Comment:
Given our target audience is developers, would it better (since shorter) to just say

`"update devurl: name must conform to regex ^[a-zA-Z][a-zA-Z0-9]{0,63}$"`
or
`"update devurl: name must be <= 64 chars and conform to regex ^[a-zA-Z][a-zA-Z0-9]"`
?